libkbfs/coalescing_context_test.go

```diff
@@ -0,0 +1,53 @@
+package libkbfs
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestCoalescingContext(t *testing.T) {
+	t.Parallel()
+	t.Log("Test basic CoalescingContext with 2 parent contexts.")
+	ctx1, cf1 := context.WithCancel(context.Background())
```

Comment:
Might as well put a value in `ctx1` and check that we can still access it in `cc`.